### ISSUE TITLE
Support explicit setting column types

### ISSUE THREAD

**Issue by rufuspollock** [okfn/webstore#43]
At the moment all column are strings. Want to support numbers etc. This would also address #25.

Two possible approaches (complementary) to setting types described in #27. This ticket is about allow setting of column type explicitly prior to load of data.

On way in all values are cast to column type and raise error if cannot cast correctly.

Question:
- Do we support types other than sql types (e.g. json "type"?)

**Comment by pudo** [comment]
Duplicate of #35, I think?

**Comment by rufuspollock** [comment]
Yes, acknowledged -- missed this because #35 is differently named :-) (and has broader discussion -- bit like #27). Key aspect there seems to be extra jargtypes parameter. Which ticket would you like to deprecate?
